### PR TITLE
Fix error: Workflow referenced an unregistered external sink

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: logging-sinks/src/worker.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 // @@@SNIPSTART typescript-logger-sink-worker
-import { Worker, InjectedSinks } from '@temporalio/worker';
-import { LoggerSinks } from './workflows';
+import { defaultSinks, InjectedSinks, LoggerSinks, Worker } from '@temporalio/worker';
+import { CustomLoggerSinks } from './workflows';
 
 async function main() {
-  const sinks: InjectedSinks<LoggerSinks> = {
+  const sinks: InjectedSinks<LoggerSinks & CustomLoggerSinks> = {
```
Comment (lorensr) [inline logging-sinks/src/worker.ts:6]: @bergundy Alternatively, SDK could export `LoggerSinks` from `@temporalio/workflow`, and then user can do 

```ts
export type AllSinks = LoggerSinks & MySinks
```

in workflow code, and use it both in workflow and here in worker code.

**Comment by lorensr** [comment]
Will compile once https://github.com/temporalio/sdk-typescript/pull/871 is merged & published

File: logging-sinks/src/worker.ts (hunk 1 above)
Comment (bergundy) [reply logging-sinks/src/worker.ts:6]: Good idea

**Review by bergundy — APPROVED** [review]
I think the instrumentation sample needs an update too

File: instrumentation/src/worker.ts [hunk 2]
```diff
@@ -50,28 +59,24 @@ async function main() {
...
-        fn(workflowInfo, message, meta) {
-          workflowLogger.child(workflowInfo).warn(message, meta);
-        },
-      },
+  const workflowLogger: Logger = {
+    log(level: LogLevel, message: string, meta?: LogMetadata) {
+      workflowWinstonLogger.log(message, meta);
+    },
+    trace(message: string, meta?: LogMetadata) {
+      workflowWinstonLogger.verbose(message, meta);
```
Comment (lorensr) [inline instrumentation/src/worker.ts:67]: no trace option

**Review by bergundy — APPROVED** [review]
Nice!

**Pull request merged**